src/utils.js

```diff
@@ -156,4 +159,13 @@ export const clamp     = (val, range)           => range ? (val < range[0] ? ran
 /**
  * Limit a delta difference between two values
  */
-export const wrapDelta = (curr, prev, deltaMax) => Math.abs(curr - prev) > deltaMax ? prev + deltaMax * Math.sign(curr - prev) : curr;
\ No newline at end of file
+export const wrapDelta = (curr, prev, deltaMax) => Math.abs(curr - prev) > deltaMax ? prev + deltaMax * Math.sign(curr - prev) : curr;
+
+/**
+ * Round numbers according to size
+ */
+export function tooltipvalue(value,count){
+	let comma = count - 1 - Math.floor(Math.log10(Math.abs(value)));
+	let factor = Math.pow(10,comma);
+	return Math.round(value * factor) / factor;
+}
```

Comment:
A new function that is hard to debug, you can include some [tests](https://github.com/Raruto/leaflet-elevation/blob/3ee745d71ed172979b7bfe0ff52f7c63214d3c04/spec/utils.test.js) to explain better your intents..

Reply:
https://leafext.info/javatest.php

Reply:
@hupe13 are you looking for something like that?

```js
let round = (x, n) => +x.toPrecision(n)
```

**Ref:** [How can I round to an arbitrary number of significant digits with JavaScript?](https://stackoverflow.com/questions/36369239/how-can-i-round-to-an-arbitrary-number-of-significant-digits-with-javascript)

Reply:
Thank you very much! This works better than my code!
https://leafext.info/javatest.php